Tests/FunctionalTests/AEPEdgeFunctionalTests.swift

```diff
@@ -58,13 +58,19 @@ class AEPEdgeFunctionalTests: TestBase {
             print("Extensions registration is complete")
             waitForRegistration.countDown()
         })
-        XCTAssertEqual(DispatchTimeoutResult.success, waitForRegistration.await(timeout: 2))
+        XCTAssertEqual(DispatchTimeoutResult.success, waitForRegistration.await(timeout: 10))
```

Comment:
I would think even 2 sec is a lot for registration, do you know aprox. how long it actually requires to run on CI?